src/views/virtualmachines/creation-wizard-new/steps/InstanceTypesSteps/ComputeResourcesStep/components/SelectInstanceTypeSection/components/RedHatProvidedInstanceTypesSection/components/RedHatInstanceTypeSeriesGallery/components/RedHatSeriesMenuCard/RedHatSeriesMenuCard.tsx

```
@@ -23,11 +25,15 @@ type RedHatSeriesMenuCardProps = {
 
-  const { selectedSeries, setSelectedSeries, setSelectedSize } = useInstanceTypeVMStore();
+  const { control, setValue } = useVMWizard();
+  const selectedSeries = useWatch({
+    control,
+    name: CREATE_VM_FORM_FIELDS_INSTANCE_TYPE_DATA.SELECTED_SERIES,
+  });
 
   const { classDisplayNameAnnotation, descriptionAnnotation, seriesName, sizes } = rhSeriesItem;
 
-  const { Icon, seriesLabel } = instanceTypeSeriesNameMapper[seriesName] || {};
+const { Icon, seriesLabel } = instanceTypeSeriesNameMapper[seriesName] || {};
```

ESLint

[error] Insert `··`

(prettier/prettier)